x-pack/plugins/maps/public/plugin.ts

```diff
@@ -65,6 +65,7 @@ import {
 import { EMSSettings } from '../common/ems_settings';
 import { SavedObjectTaggingPluginStart } from '../../saved_objects_tagging/public';
 import { ChartsPluginStart } from '../../../../src/plugins/charts/public';
+import { suggestEMSTermJoinConfig } from './api/register';
```

Comment:
Can you move import to api import `import { createLayerDescriptors, registerLayerWizard, registerSource } from './api';`